client/my-sites/plugins/controller.js

```diff
@@ -204,3 +205,8 @@ export function scrollTopIfNoHash( context, next ) {
 	}
 	next();
 }
+
+export function renderDomainsPage( context, next ) {
+	context.primary = <MarketplaceDomainUpsell />;
```

Comment:
Seems like this is inflating the plugins section by a huge percent (38.3% gzipped [according to ICFY](https://github.com/Automattic/wp-calypso/pull/52081#issuecomment-822971579)). Is this something we can try to `AsyncLoad`? Or maybe we can `AsyncLoad` pieces in it that are responsible for the inflation, like the domain picker for example?